debian/postinst

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function die
+{
+    echo "$1" >&2
+    exit 1
+}
+
+target="/boot/recovery.img"
+workdir=$(mktemp -d)
+
+cd $workdir
+sudo chown root .
```

Comment:
Similar to another comment I left, do we need to use `sudo` here? IIRC, we'll run this as root via the post-install package hook, so I think the use of `sudo` can be removed (and should be removed, if it's unnecessary, IMO).